Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -30,11 +34,22 @@ @implementation WMFArticlePreviewTableViewCell
 
 #pragma mark - Setup
 
+- (void)blurAndShowLoadingIndicator {
```

Comment:
@montehurd what i've done in these cases is just expose a setter method (no `@property`), since "getting" the value doesn't really make sense (nor is it usually necessary).